tests/pos-macros/quote-sym-newboundedtype/Macro_1.scala

```diff
@@ -0,0 +1,22 @@
+//> using options -experimental -Yno-experimental
+import scala.quoted.*
+
+inline def testMacro = ${ testImpl }
+
+transparent inline def transparentTestMacro = ${ testImpl }
+
+def testImpl(using Quotes): Expr[Object] = {
+    import quotes.reflect.*
+
+    def makeType(owner: Symbol): Symbol =
+      Symbol.newBoundedType(owner, "mytype", Flags.EmptyFlags, TypeBounds.lower(TypeRepr.of[String]), Symbol.noSymbol)
+
+    val typeDef = TypeDef(makeType(Symbol.spliceOwner))
+    // Expr printer does not work here, see comment:
```

Comment:
If this cannot be fixed right now, we should open an issue for it. (IMO we should just use the regular compiler type printer and get rid of the custom one used for quotes, it creates more problems than it solves, and we can make the regular type printer more flexible if needed).